### ISSUE TITLE
ttf-linux-libertine is going away from repos...

### ISSUE THREAD

**Issue by piyushgarg** [LukeSmithxyz/LARBS#516]
`(64/69) upgrading ttf-linux-libertine              [----------------] 100%

Note: development of Libertine seems to have halted, despite outstanding bugs.
Users are advised to upgrade to the de facto successor,
[community]/libertinus-font, and to update any documents and configuration
if necessary.
For compatibility's sake, Linux Libertine will remain in the repos until 2022.
Please refer to https://wikipedia.org/wiki/Linux_Libertine#Derivative_works
for more information.`

**Comment by kronikpillow** [comment]
yes I tried to replace ttf-linux-libertine with libertinus font a long time ago with a PR, but Luke for some reason declined it
